components/layout/AuthLayout.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, useContext } from "react";
+import Head from "next/head";
+import { tw } from "../../utils";
+import { UIContext } from "../../context/ui";
+import { DarkModeIcon, LightModeIcon } from "../icons/Icons";
+
+interface HomeLayoutProps {
+  title: string;
+  pageDescription: string;
+  children: React.ReactNode;
+}
+
+export const AuthLayout: FC<HomeLayoutProps> = ({
+  title,
+  pageDescription,
+  children,
+}) => {
+  const { darkMode, activateDarkMode, deactivateDarkMode } =
+    useContext(UIContext);
+
+  return (
+    <div className={tw("h-screen ", darkMode ? "dark" : "")}>
+      <Head>
+        <title>{title}</title>
+        <meta name="description" content={pageDescription} />
+      </Head>
+
+      <main className="flex h-full">
+        <div
+          className="flex h-full w-full justify-end bg-cover bg-center"
+          style={{
+            backgroundImage:
+              "url(https://healthfox.fi/en/wp-content/uploads/2022/12/NHS_doctor-scaled.jpg)",
```

Comment:
you shouldn't be linking to another website to use as a background image. it's fine for now but you should probably download the image and look into how nextjs handles public images